src/hooks/useMeritIncentives.ts

```diff
@@ -124,7 +124,7 @@ const MERIT_DATA_MAP: Record<string, Record<string, MeritReserveIncentiveData[]>
       {
         action: MeritAction.ETHEREUM_SGHO,
         rewardTokenAddress: AaveV3Ethereum.ASSETS.GHO.UNDERLYING,
```

Comment:
We should use `AaveSafetyModule.STK_GHO` (former stkGHO <> current sGHO) instead of `AaveV3Ethereum.ASSETS.GHO.UNDERLYING`

Reply:
yeah missed that one thanks updated in https://github.com/aave/interface/commit/399852a66ae485abb64290861328b6415916832f

Reply:
already fixed in https://github.com/aave/interface/commit/399852a66ae485abb64290861328b6415916832f